setup.py

```diff
@@ -14,6 +14,9 @@
         'iotile_cloud.utils',
         'iotile_cloud.stream'
     ],
+    entry_points={
+        'pytest11': ['iotile-analytics = iotile_cloud.utils.mock_cloud']
```

Comment:
Is this line correct? `iotile-analytics`?

Reply:
sorry, shouldn't affect anything but yes an oversight.